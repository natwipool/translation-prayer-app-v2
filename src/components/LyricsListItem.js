import React from 'react';

const LyricsListItem = ({ lyric }) => (
  <div>
    <p>{lyric}</p>
  </div>
)

export default LyricsListItem;
