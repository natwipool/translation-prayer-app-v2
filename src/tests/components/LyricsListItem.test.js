import React from 'react';
import { shallow } from 'enzyme';
import LyricsListItem from '../../components/LyricsListItem';
import transPrayers from '../fixtures/transPrayers';

test('should render LyricsListItem correctly', () => {
  const wrapper = shallow(<LyricsListItem lyric={transPrayers[0].lyrics[1]}/>)
  expect(wrapper).toMatchSnapshot();
});