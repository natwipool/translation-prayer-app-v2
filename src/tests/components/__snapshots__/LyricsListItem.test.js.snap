// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LyricsListItem correctly 1`] = `
<div>
  <p>
    dfg
  </p>
</div>
`;
